src/kili/queries/asset/media_downloader.py

```diff
@@ -26,7 +28,17 @@ def get_download_assets_function(
     """
     if not download_media:
         return None
-    input_type = get_project_field(kili, project_id, "inputType")
+    projects = list(
+        ProjectQuery(kili.auth.client)(
+            ProjectWhere(project_id=project_id), fields, QueryOptions(disable_tqdm=True)
+        )
+    )
+    if len(projects) == 0:
+        raise NotFound(
+            f"project ID: {project_id}. Maybe your KILI_API_KEY does not belong to a member of the"
+            " project."
+        )
```

Comment:
aren't we duplicating the code of get_project in src/kili/services/project/__init__.py ?

Reply:
we should not call a service from a core Class (if we do, this should be changed)

Reply:
yes, I duplicated the code but because we had a circular import.
I had it several time for small modifications so I think that It is better to not call the service layer here. Or we can take a ticket to better decorrelate the media_downloader, the services and the queries